# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webmessaging/broadcastchannel/detached-iframe.html.ini
# Commit: b1ab6c750d04
# Full Hash: b1ab6c750d04299011c1446b1752c2ff98a92b33
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-06-07 03:30:33
# Regressor Bug: 1800659
# File Overlap Count: 1
# Description:
#   Bug 1800659 - P3 Remove WorkerPrivate::ClearMainEventQueue() r=asuth,smaug
#   
#   Pending->Canceling->Killing (WorkerNeverRan)
#   
#   Need to clear WorkerPrivate::mPreStartRunnables
# ==============================================================================

diff -r 32515f9b41c7 -r b1ab6c750d04 testing/web-platform/meta/webmessaging/broadcastchannel/detached-iframe.html.ini
--- a/testing/web-platform/meta/webmessaging/broadcastchannel/detached-iframe.html.ini	Tue Jun 06 06:36:50 2023 +0000
+++ b/testing/web-platform/meta/webmessaging/broadcastchannel/detached-iframe.html.ini	Tue Jun 06 06:36:50 2023 +0000
@@ -4,9 +4,6 @@
   [BroadcastChannel messages from detached iframe to parent should be ignored (BC created before detaching)]
     expected: FAIL
 
-  [BroadcastChannel messages from detached iframe to parent should be ignored (BC created after detaching)]
-    expected: FAIL
-
   [BroadcastChannel messages within detached iframe should be ignored (BCs created before detaching)]
     expected: FAIL
 